Tibia/MonsterRepository.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Tibia
+{
+    public class MonsterRepository
```

Comment:
To powinno tylko trzymać listę potworów i get potwora nie powinien być równoważny z zabiciem potwora